sdk/js/index.ts

```diff
@@ -167,32 +260,34 @@ export default class StreamManager {
       throw new Error("Error while processing trasactions");
 
```

Comment:
can do `await publicClient.waitForTransactionReceipt(hash)` to wait for transaction to fully process

Reply:
we are moving this to another PR